### PR TITLE
Issue 970 - add Bogota workshop to /research

### PR DESCRIPTION
**Pull request by acrymble**
I have added our Bogota workshop to the English and Spanish versions of the /research page:

* Adam Crymble, Maria José Afanador Llach, José Antonio Motilla, 'Digital Humanities Skills for Latin American Scholarship', Universidad de los Andes, Bogotá, Colombia (August 2018) - funded by the British Academy.

Please check my Spanish. And @spapastamkou can you please add this to the French page? I am not sure where that is being kept. This should appear at the top of the "Workshops" section.

### CODE REVIEW

File: es/investigacion.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ El equipo del proyecto y los miembros de la comunidad en general están involucr
 
 ### Talleres y eventos
 
-* Victor Gayol, 'Introducción a las Ciencias Sociales y Humanidades Digitales. The Programming Historian en español – COLMICH', El Colegio de Michoacán (septiembre 2017)
+* Adam Crymble, Maria José Afanador Llach, José Antonio Motilla, 'Digital Humanities Skills for Latin American Scholarship', Universidad de los Andes, Bogotá, Colombia (agosto 2018) - financiado por la British Academy.
```
Comment (arojascastro) [inline es/investigacion.md]: Maybe we can add the days... so we keep the same format.

**Comment by spapastamkou** [comment]
Sure. The page in FR is not created yet but I started working on the file
and will let you know when I make the commit.

File: es/investigacion.md (hunk 1 above)
Comment (acrymble) [reply es/investigacion.md]: I've added the dates to the workshop in Bogota. I don't know what they were for Victor's workshop last year.

**Comment by spapastamkou** [comment]
all updates ok in FR

**Comment by acrymble** [comment]
This requires re-basing. @walshbr I don't suppose you could help? *sheepish look*

**Comment by walshbr** [comment]
Done @acrymble! Checks are good, so you should be able to merge once all the translation stuff is set. Looks like the ES conservation was still ongoing by my quick read of your back and forth with @arojascastro, but you'd know best.

**Comment by acrymble** [comment]
I'm still waiting for a review for this please.

**Review by walshbr — APPROVED** [review]
The English at least looks fine.

**Comment by acrymble** [comment]
Thanks @arojascastro

**Pull request merged**